xwiki-rendering-api/src/main/java/org/xwiki/rendering/syntax/SyntaxType.java

```diff
@@ -239,10 +239,10 @@ public boolean equals(Object object)
                 result = false;
             } else {
                 // Object must be Syntax at this point.
-                SyntaxType syntaxType = (SyntaxType) object;
+                SyntaxType syntaxType = SyntaxType object;
                 // Note that the name isn't part of the hashCode computation since it's not part of the Syntax type's
                 // identity.
-                result = (getId() == syntaxType.getId() || (getId() != null && getId().equals(syntaxType.getId())));
+                result = (getId() == syntaxType.getId() || (getId() != null && getId().equalssyntaxType.getId()));
```

Comment:
This also can't work... Please don't create PRs that don't pass the build ;)

Reply:
sorry :)